main.go

```diff
@@ -302,17 +301,7 @@ func (cmd *mainCmd) Run(args []string) error {
 			continue
 		}
 
-		bs, err := imports.Process(filename, out.Bytes(), &imports.Options{
```

Comment:
Should this be removed completely or configurable behind some type of option?

Reply:
I looked into that but sadly it doesn't have that option
Process can be configured by the [options](https://pkg.go.dev/golang.org/x/tools/imports#Options) struct.
When `FormatOnly` is `true` it only formats the imports and when it is set to `False` it formats and modifies the import statements.  
Either case, it ends up apply the formatting rule which would have the groupinf case

Reply:
Correct me if I misunderstood you @r-hang but I think what he's trying to say is that this is hidden behind an option that's exposed at gopatch level, not the `import.Process`. 

i.e. Should we disable group import ordering if we pass --disable-group-imports flag to gopatch.

Reply:
Ohh My bad. Yeah we can definitely add a flag for this. 
Though, technically these formatting rules are being applied to non patch affected code through the `import.Process`
call. 
Should gopatch's scope be to only modify patch matching code (remove the call altogether)
 or 
also have formatting for the non-matching imports ( we add a  --disable-group-imports flag)
The 2nd option would be cleaner & also follow go's grouping order